scubagoggles/reporter/reporter.py

```diff
@@ -317,8 +319,13 @@ def rego_json_to_ind_reports(test_results_data : str, product : list, out_path :
                                 'Result': result,
                                 'Criticality': test['Criticality'],
                                 'Details': details})
+        markdown_group_name = "-".join(baseline_group['GroupName'].split())
+        group_reference_url = f'<a href="{SCUBA_GITHUB_URL}/tree/main/baselines/'\
```

Comment:
ScubaGear links to the version of the baseline that generated the report, not to main. We should probably maintain that pattern here for consistency.

Unfortunately we don't yet have a variable or anything that stores the ScubaGoggles version, so for now just hard-coding the version if fine, like this:
```suggestion
        group_reference_url = f'<a href="{SCUBA_GITHUB_URL}/tree/v0.2.0/baselines/'\
```